bioimageio/spec/__main__.py

```diff
@@ -1,96 +1,64 @@
-from copy import deepcopy
+import warnings
 from pathlib import Path
 from pprint import pprint
 
-import requests
 import typer
-from marshmallow import ValidationError
 from ruamel.yaml import YAML
 
-from bioimageio.spec import maybe_convert_model, maybe_convert_manifest, schema
+from bioimageio.spec import load_raw_node
 
 yaml = YAML(typ="safe")
 
 app = typer.Typer()  # https://typer.tiangolo.com/
 
 
 @app.command()
-def package_model(model_yaml: Path, auto_convert: bool = False):
-    pass
+def package(rdf_source: str, auto_convert: bool = False):
```

Comment:
I guess this is being implemented in #147?

Reply:
the backend is, just missing compression (https://github.com/bioimage-io/spec-bioimage-io/pull/146#discussion_r662778485)

I'll make another PR to #146 for this to keep them small

Reply:
opened #152